javascript-constructors/student.js

```diff
@@ -0,0 +1,18 @@
+/* exported Student */
+function Student(firstName, lastName, subject) {
+  this.firstName = firstName;
+  this.lastName = lastName;
+  this.subject = subject;
+}
+
+Student.prototype.getFullName = function () {
+  var fullName = this.firstName + ' ' + this.lastName;
+  return fullName;
+};
+
+Student.prototype.getIntroduction = function () {
+  var student = new Student(this.firstName, this.lastName, this.subject);
+  var personFullName = student.getFullName();
```

Comment:
You shouldn't be making a new Student object in here, you want to work off of the Student object this method is attached to and being called from, which would be `this`

Reply:
So essentially var personFullName = this.getFullName();